install/python-installer/traf_setup.py

```diff
@@ -50,6 +50,9 @@ def run():
     ### kernel settings ###
     run_cmd('sysctl -w kernel.pid_max=65535 2>&1 > /dev/null')
     run_cmd('echo "kernel.pid_max=65535" >> /etc/sysctl.conf')
+    run_cmd('cp $SQ_ROOT/sysinstall/etc/init.d/trafodion /etc/init.d')
```

Comment:
Good point. I would leave it as is till /etc/trafodion/trafodion_config is available.  The service script needs to be executed by root as part of shutdown.  The variable SQ_ROOT will not be available to root user unless the service script trafodion is regenerated on the fly after installation or the new installer stores some default info at /etc/default/trafodion

Also, by leaving it as it is possible for the existing installation to configure trafodion as a service by issuing the commands in the traf_setup.py manually